src/components/SuggestedJourneys/SuggestedJourneys.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { stationsById } from "stations";
+import { JourneyParams } from "types";
+
+import styles from "./SuggestedJourneys.module.scss";
+
+interface journey {
+    // origin: string;
+    // destination: string;
+    // originStationId: JourneyParams["fromStationId"];
+    // destStationId: JourneyParams["toStationId"];
+    // day: JourneyParams["day"];
+    // time: JourneyParams["time"];
+    params: JourneyParams;
+    fromStationName: string;
+    toStationName: string;
+}
+
+type Props = { 
+    journeys: journey[];
+}
+
+export function getJourneyUrl(fromStationId: string, toStationId: string, day: string, time: number | undefined) {
+    return `/?from=${fromStationId}&to=${toStationId}&day=${day}&time=${time}`;
+}
+
+const SuggestedJourneys = (props: Props) => {
+    const { journeys } = props;
+    let divArr = [] as any;
+    for (let i = 0; i < journeys.length; ++i) {
+        divArr.push(
+            <div className={styles.journey}> 
+                 <a href={getJourneyUrl(journeys[i].params["fromStationId"], journeys[i].params["toStationId"], journeys[i].params["day"], journeys[i].params["time"])}>
+                 <h2>Journey Option {i + 1}:</h2>
+                 <p>Depart from: { journeys[i].fromStationName }</p>
```

Comment:
Since you reference `journeys[i]` several times it might be cleaner to write:

```
const journey = journeys[i]
```

or something like that at the beginning of the loop.